docs/admonitions.rst

```diff
@@ -0,0 +1,230 @@
+
+Admonitions
+===========
+
+The theme uses the ``admonition`` directives for Sphinx admonitions.
+
+rST and Sphinx
+**************
+
+Most of the admonitions that the mkdocs-material theme supports were "borrowed" from
+admonitions defined in the reStructuredText specifications. You may recognize them from
+usage in other sphinx-based themes. They are:
+
+``note``, ``todo``, ``seealso``
+   .. seealso::
+      This admonition is specific to Sphinx directives and not defined in the rST specifications
+      as you can `seealso`.
+
+      ``note`` and ``todo`` are admonitions defined by the rST specifications.
+
+``tip``, ``hint``, ``important``
+   .. important::
+      It is **important** to correctly use admonitions.
+
+``attention``, ``caution``, ``warning``
+   .. warning::
+      This is a **warning**.
+
+``danger``, ``error``
+   .. error::
+      You have made a grave **error**.
+
+Admonitions from mkdocs-material
+********************************
+
+Some addtional admonitions are supported via the source code from the mkdocs-material theme.
+These admonitions can still be used, but the syntax is a little different because it relies
+on the generic admonition defined in the reStructuredText specifications.
+
+To use the following admonitions' styles from the mkdocs-material theme, the rST syntax is
+shown inside the demonstrated admonition.
+
+.. important::
+   The ``:class:`` options below (in the rST code blocks) must use lower case letters for the
+   styling to work. Otherwise, the admonition will look like a ``note`` (as that is the
+   default fallback style).
+
+``abstract``, ``summary``, ``tldr``
```

Comment:
We could define custom sphinx directives for all of these admonitions, though I'm not sure how useful that would be compared to just specifying `:class`.  Some of the names may conflict with existing sphinx directives.

Reply:
After trying to understand the docutils module, I have to err with using the `class `option. Additionally, using the `class` option also helps mimic the mkdocs-material feature to customize a user-defined admonition (as shown in the bottom of this new docs page).